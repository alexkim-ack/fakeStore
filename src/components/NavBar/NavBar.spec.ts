import { expect, describe, it } from "vitest";

import { NavBar } from "./index";

describe("Navbar", () => {
    it("should render", () => {
        const render = NavBar();
        expect(render).toMatchSnapshot();
    });
});
