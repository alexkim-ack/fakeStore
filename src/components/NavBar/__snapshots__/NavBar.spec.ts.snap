// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar > should render 1`] = `
<nav
  className="border-b-4 border-indigo-500 bg-indigo-500"
>
  <ul
    className="list-none"
  >
    <li>
      <Link
        to="/"
      >
        <p
          className="text-white"
        >
          Home
        </p>
      </Link>
    </li>
  </ul>
</nav>
`;
